tests/flows/erc20_source_multihop.go

```diff
@@ -0,0 +1,164 @@
+package flows
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	erc20source "github.com/ava-labs/teleporter-token-bridge/abi-bindings/go/ERC20Source"
+	"github.com/ava-labs/teleporter-token-bridge/tests/utils"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	teleporterUtils "github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	. "github.com/onsi/gomega"
+)
+
+func ERC20SourceMultihop(network interfaces.Network) {
+	/**
+	 * Deploy a erc20 token source on the primary network
+	 * Deploys ERC20Destination to Subnet A and Subnet B
+	 * Bridges C-chain example erc20 tokens to Subnet A
+	 * Bridge tokens from Subnet A to Subnet B through multihop
+	 * Brige back tokens from Subnet B to Subnet A through multihop
+	 */
```

Comment:
```suggestion
	/**
	 * Deploy a ERC20 token source on the primary network
	 * Deploys ERC20Destination to Subnet A and Subnet B
	 * Bridges C-Chain example ERC20 tokens to Subnet A
	 * Bridge tokens from Subnet A to Subnet B through multihop
	 * Bridge back tokens from Subnet B to Subnet A through multihop
	 */
```

Also would prefer it to be above the function definition.